pkg/controller/knativeserving/openshift/openshift.go

```diff
@@ -85,6 +87,45 @@ func Configure(c client.Client, s *runtime.Scheme, manifest *mf.Manifest) (*comm
 	return &extension, nil
 }
 
+func CheckVersion(c client.Client, s *runtime.Scheme, instance *servingv1alpha1.KnativeServing) (bool, error) {
+	minVersion := semver.New("4.1.12")
```

Comment:
The right minimum version here is not 4.1.12. It's currently schedule for 4.1.13.

Reply:
Updated.